taxonomy/models.py

```diff
@@ -357,3 +357,84 @@ def __repr__(self):
             'unique_postings="{4!r} unique_companies={5!r}">'.format(
                 self.id, self.job, self.median_salary, self.median_posting_duration, self.unique_postings,
                 self.unique_companies)
+
+
+class Translations(TimeStampedModel):
+    """
+    Model to save translated descriptions.
+
+    .. no_pii:
+    """
+
+    source_model_name = models.CharField(
+        max_length=255,
+        help_text=_(
+            'The name of the model to which the source belongs e:g Course.'
+        )
+    )
+
+    source_model_field = models.CharField(
+        max_length=255,
+        help_text=_(
+            'The name of the source field to be translated e:g course description.'
+        )
+    )
+
+    source_record_identifier = models.CharField(
+        max_length=255,
+        unique=True,
```

Comment:
It would be better if we add unique together based on 3 fields (source_model_name, source_model_field, source_record_identifier)

Reply:
Done.